spec/views/report/_form_expression_buttons.html.haml_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe 'report/_form_filter.html.haml' do
+  let(:report) { FactoryGirl.create(:miq_report) }
+  let(:exp_table) { [["Example AFTER \"value1\"", 1]] }
+  let(:expression) { {"after" => {"field" => "field_example", "value" => "value1"}, :token => 1} }
+  let(:edit) { { :rpt_id => report.id, :record_filter => {:exp_table => exp_table, :expression => expression}} }
+
+  before do
+    assign(:expkey, :record_filter)
```

Comment:
you missed this :wilted_flower:

Reply:
I don't get it.. what exactly? 👼

Reply:
:scissors: :skull: deleting this line

Reply:
```
before { assign(:edit, edit) }

Reply:
really? I don't understand. If I do it, @edit will be nil and the test will always fail.

Reply:
Delete the `expkey` line and keep the one with the `edit`, preferred in a one-liner `before` block I commented above instead of a `do-end` one.

Reply:
I still don't understand. maybe we need more talk about this somewhere else. Because if I delete `assign(:expkey, :record_filter)` line, @expkey in haml will be nil and again, the spec will always fail. We need it there because of @expkey in this line

Reply:
aarh yes, my bad, i'm wrong ... sorry :cry: